docs/Tutorials/Visualization.md

```diff
@@ -2,26 +2,74 @@
+Each SpECTRE executable is paired with a user-provided YAML file that
+specifies the runtime settings of the evolution. This is where information on
+options such as the Domain, AnalyticSolution, TimeStepper, etc is supplied.
+Example input files are kept in `tests/InputFiles`, and are written to provide
+a functional base input file from which the user can then modify to as desired.
+The YAML file is passed as an argument to the executable using the flag
+`--input-file`. For example, for a scalar wave evolution, run the command:
+`./EvolveScalarWave3D --input-file Input3DPeriodic.yaml`, where
+`Input3DPeriodic.yaml` was copied over from the `InputFiles` directory to the
+`bin` directory. A successful evolution will result in the creation of two H5
+files whose names can be specified in the YAML file under the options
+`VolumeFileName` and `ReductionFileName`. Visualization of the volume data will
+be described in the next section.
+
+### 3D %Data Volume %Data In ParaView
+
+A SpECTRE executable produces a volume data h5 file and a reduced data h5 file.
+An XDMF file must be created from the volume data in order to do
+visualization using ParaView. To this end we provide the python function
+`GenerateXdmf.py` in the `tools` directory. The `--file-prefix` and `--output`
+arguments must be specified when the command is run; they correspond to the
+names of the H5 volume data file and the desired .xmf file name, respectively,
+and without filename extensions. Open the .xmf file in ParaView
+and select the `Xdmf Reader`, *not* the version 3 readers. On the left hand side
 of the main ParaView window is a section named `Properties`, here you must
 click the highlighted `Apply` button. ParaView will now render your volume data.
+
+### Helpful ParaView Filters
+
+Here we describe the usage of filters we've found to better visualize our data.
+Feel free to contribute to this section!
+
+#### Removing Mesh Imprinting
 You may notice what appears to be mesh imprinting on the data. The imprinting
 effect can be removed by applying the `Tetrahedralize` filter. To apply the
 filter select the `Filters` menu item, then `Alphabetical` and finally
 `Tetrahedralize`.
 
+#### Creating Derived Volume Data
+New volume data can be created from the existing volume data using the
+`Calculator` filter. In the `Calculator`'s text box, input a numerical
+expression in terms of existing datasets evaluating to the desired
+quantity. For example, a vector-valued velocity dataset can be created
+from three scalar velocity component datasets using the expression
+`velocity_x * iHat + velocity_y * jHat + velocity_z * zHat` and hitting
+the `Apply` button. By default, this will create a new dataset `Result`,
+which can be changed by changing the name provided to `Result Array Name`
+above the `Calculator` text box.
+
+#### Visualizing Vector Fields Derived From Scalar Fields
+Use the `Calculator` filter described above to create a new vector-valued
```

Comment:
You should mention the section name, since if there are a lot more filters added "above" could be vague